### PR TITLE
Created endpoint for changing user password

### PR DESCRIPTION
**Pull request by MishaAfanasiuk**
**Description:** Created endpoint for changing user password

**Link to Trello ticket:** https://trello.com/c/ORvlLLnb

### CODE REVIEW

File: src/modules/users/dto/newPassword.dto.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty, IsString, Length, MinLength } from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class NewPasswordDto {
+  @IsNotEmpty()
+  @Type(() => String)
```
Comment (MishaHnatyshyn) [inline src/modules/users/dto/newPassword.dto.ts]: Why do we need this here?

File: src/modules/users/dto/newPassword.dto.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  @MinLength(6)
+  @IsString()
+  old_password: string;
+
+  @IsNotEmpty()
+  @Type(() => String)
```
Comment (MishaHnatyshyn) [inline src/modules/users/dto/newPassword.dto.ts]: Same here

File: src/modules/users/dto/newPassword.dto.ts (hunk 1 above)
Comment (MishaAfanasiuk) [reply src/modules/users/dto/newPassword.dto.ts]: Somebody will send 123456

File: src/modules/users/dto/newPassword.dto.ts (hunk 2 above)
Comment (MishaAfanasiuk) [reply src/modules/users/dto/newPassword.dto.ts]: Same here

File: src/modules/users/dto/newPassword.dto.ts (hunk 1 above)
Comment (MishaHnatyshyn) [reply src/modules/users/dto/newPassword.dto.ts]: if someone sends 123456, then request body is not valid and the sender will receive an error

Comment (MishaAfanasiuk) [reply src/modules/users/dto/newPassword.dto.ts]: But we can just change password (

**Review by MishaHnatyshyn — APPROVED** [review]

**Pull request merged**